src/main/java/teammates/common/datatransfer/questions/FeedbackContributionQuestionDetails.java

```diff
@@ -44,21 +44,27 @@ public class FeedbackContributionQuestionDetails extends FeedbackQuestionDetails
 
     private static final Logger log = Logger.getLogger();
 
+    private Boolean isZeroSum;
     private boolean isNotSureAllowed;
 
     public FeedbackContributionQuestionDetails() {
-        this(null);
+        super(FeedbackQuestionType.CONTRIB, null);
+        // isZeroSum is not set by default to differentiate between old and new contribution questions,
+        // i.e., old questions do not have the isZeroSum field while new questions do.
```

Comment:
> We set the default value of isNotSureAllowed to false here so that the default value if isZeroSum is not present will be false.

I got a little confused here. I thought the reason for the default value of `isNotSureAllowed` to be `false` is that that's the new default value for `isNotSureAllowed` with the addition of `isZeroSum`, no?